app/src/main/res/layout/content_main_l.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/get_content_card"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+  <androidx.cardview.widget.CardView
+      android:id="@+id/content_main_card"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="8dp"
+      app:cardCornerRadius="4dp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      >
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/constraint_main"
+        android:background="@drawable/back_cover">
+
+      <ImageView
+          android:id="@+id/content_side_image"
+          android:layout_width="0dp"
+          android:layout_height="144dp"
+          android:scaleType="fitEnd"
+          android:contentDescription="@string/side_design"
```

Comment:
`@string/side_design` is assigned `sideDesign`, as the `contentDescription is used by screen readers you should either give it a meaningful string that can we be understood by a person when it is read to him/her or you should set it to "@null" for elements that are only for decoration purposes. You should read this full heading: https://developer.android.com/guide/topics/ui/accessibility/apps#label-elements. In this case, you must use "@null".